src/makeCalculator.js

```diff
@@ -4,7 +4,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate: function (sv, x) {
+      this.result = sv(this.result, x);
+
+      return this;
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+    multiply: (result, x) => result * x,
```

Comment:
The `multiply` method should not be an arrow function as it does not have its own `this` context, which will cause issues when trying to access the calculator's `result` property. Use a regular function instead.